tests/ftests/systemd.py

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Systemd class for the libcgroup functional tests
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+from run import Run, RunError
+
```

Comment:
Python expects two empty lines between imports and class definition.

Reply:
D'oh. I thought we had checks in place for this.  Good catch.

Reply:
We had them as part of the `libcgroup-test` repo and after the movement of it, into the `libcgroup` repo we did not include the `lint` check in the `libcgroup` Github actions/workflow. I have introduced it in the `libcgroup` Github actions/workflow, please have a look at #266 and #267 and let me know your thoughts.